lib/Capabilities.php

```diff
@@ -293,40 +295,25 @@ public function getCapabilities(): array {
 			$capabilities['config']['signaling']['hello-v2-token-key'] = $pubKey;
 		}
 
-		/** @var ?list<string> $predefinedBackgrounds */
-		$predefinedBackgrounds = null;
-		$cachedPredefinedBackgrounds = $this->talkCache->get('predefined_backgrounds');
-		if ($cachedPredefinedBackgrounds !== null) {
-			// Try using cached value
-			/** @var list<string>|null $predefinedBackgrounds */
-			$predefinedBackgrounds = json_decode($cachedPredefinedBackgrounds, true);
-		}
+		$includeBrandedBackgrounds = $user instanceof IUser || $this->appConfig->getAppValueBool('backgrounds_branded_for_guests');
+		$includeDefaultBackgrounds = !$user instanceof IUser || $this->appConfig->getAppValueBool('backgrounds_default_for_users', true);
```

Comment:
Shouldn't that affect everyone?
I would assume, that if my local users can't use them, neither can guests (then if no branded, only blur will be allowed)

Reply:
Well the idea in the coworking call was:
- Branded should be only for company internal users, so only they "look official"
- Guests should still have an option to hide more and not have to reveal the blurred version of their live.